md-rest-api/src/main/java/com/wipro/ats/bdre/md/rest/ProcessAPI.java

```diff
@@ -209,17 +227,22 @@ RestWrapper list(@RequestParam(value = "page", defaultValue = "0") int startPage
      * @return restWrapper It contains the updated instance of Process.
      */
     @RequestMapping(value = {"/", ""}, method = RequestMethod.POST)
-    @ResponseBody public
-    RestWrapper update(@ModelAttribute("process")
-                       @Valid Process process, BindingResult bindingResult, Principal principal) {
+    @ResponseBody
+    public RestWrapper update(@ModelAttribute("process")
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) The Cyclomatic Complexity of this method "update" is 16 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)